hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -31,6 +33,7 @@
 #include "hardware_interface/types/lifecycle_state_names.hpp"
 #include "lifecycle_msgs/msg/state.hpp"
 #include "rclcpp/duration.hpp"
+#include "rclcpp/executor.hpp"
```

Comment:
Why do we need an executor here? This is probably just left-over from development.

Reply:
removed